M2/Macaulay2/tests/ComputationsBook/programming/chapter.out.expected

```diff
@@ -269,7 +269,7 @@ o64 : Matrix R  <--- R
 o65 = matrix {{x, y, z}}
 i66 : toExternalString vars R
 
-o66 = map(R^{{0}}, R^{{-1}, {-1}, {-1}}, {{x, y, z}})
+o66 = map(R^1,R^{{-1}, {-1}, {-1}},{{x, y, z}})
```

Comment:
@pzinn This is why I asked.

Reply:
I'm not sure how up to date some of these "expected" files are.

Reply:
Oh, I see. they do get checked regularly, but with `test.out.expected`, not `chapter.out.expected`. You should change the former only, I believe.

Reply:
As far as I understand the only one that's tested is test.out.expected?
I actually fixed the other one (chapter.out.expected) by mistake.

Reply:
OK. undone.